sdk/python/setup.py

```diff
@@ -38,7 +38,7 @@
     'Deprecated',
     'strip-hints',
     'docstring-parser>=0.7.3',
-    'kfp-pipeline-spec>=0.1.0, <0.2.0',
+    'kfp-pipeline-spec>=0.1.0, <0.2.0, !=0.1.3',
```

Comment:
This would require a release of `kfp`. How about we just delete kfp-pipeline-spec 0.1.3 from pypi?

Reply:
Already deleted.

That said I think we can still leave `!=0.1.3` here just to be on the safer side. WDYT?

Reply:
Not really needed, once deleted from PyPi, the version becomes non-discoverable (there might be a few minutes delay).
That said, it's fine to leave it as-is. We can clean it up the next time we touch this file.

/lgtm

Reply:
Make sense. I removed the `!=0.1.3`